backend/open_webui/migrations/versions/29c6dc90b192_create_stripe_payment_history_table.py

```
@@ -0,0 +1,42 @@
+"""Create stripe_payment_history table
+
+Revision ID: 29c6dc90b192
+Revises: 7bd38c980997
+Create Date: 2025-02-19 12:29:28.487440
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import open_webui.internal.db
```

Ruff (0.8.2)

`open_webui.internal.db` imported but unused

Remove unused import: `open_webui.internal.db`

(F401)